core/config/project_settings.cpp

```diff
@@ -469,6 +469,12 @@ bool ProjectSettings::_load_resource_pack(const String &p_pack, bool p_replace_f
 		return false;
 	}
 
+	// this pack may have declared new global classes (make sure they are picked up)
```

Comment:
```suggestion
	// This pack may have declared new global classes (make sure they are picked up).
```
Make sure to apply this style to all comments (starts with a capital letter, ends with a dot).